src/formio/components/FileField.js

```diff
@@ -0,0 +1,16 @@
+import {Formio} from 'react-formio';
+
+import {applyPrefix} from '../utils';
+
+
+class FileField extends Formio.Components.components.file {
+  get inputInfo() {
+    const info = super.inputInfo;
+    // change the default CSS classes
+    info.attr.class = applyPrefix('file');
+    return info;
+  }
+}
```

Comment:
`inputInfo` is not used by the file parent component, since it doesn't inherit from `Input/MultiValue`. So this entire subclass doesn't really have a reason to exist and the default widget should do instead.